Maturin 0.14.0 and later requires Python package metadata be in pyproject.toml
rather than Cargo.toml or the package will fail to build.

diff -u a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,15 +9,6 @@
 homepage = "https://github.com/ArniDagur/python-adblock"
 repository = "https://github.com/ArniDagur/python-adblock"
 
-[package.metadata.maturin]
-classifier = [
-    "Programming Language :: Python",
-    "Programming Language :: Rust",
-    "License :: OSI Approved :: MIT License",
-    "License :: OSI Approved :: Apache Software License",
-]
-requires-python = ">=3.7"
-
 [profile.release]
 debug = true
 
diff -u a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -15,3 +15,22 @@
 [build-system]
 requires = ["maturin>=0.12,<0.13"]
 build-backend = "maturin"
+
+[project]
+name = "adblock"
+version = "0.6.0"
+authors = [
+  { name="Ãrni Dagur", email="arni@dagur.eu" },
+]
+readme = "README.md"
+repository = "https://github.com/ArniDagur/python-adblock"
+classifier = [
+    "Programming Language :: Python",
+    "Programming Language :: Rust",
+    "License :: OSI Approved :: MIT License",
+    "License :: OSI Approved :: Apache Software License",
+]
+requires-python = ">=3.7"
+
+[project.urls]
+"Source Code" = "https://github.com/ArniDagur/python-adblock"
